src/Components/Field/Field.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Field.css'
+
+export default ({
+    plantSeeds,
+    seedTypes,
+    field,
+    harvest
+}) =>
+    <div data-is="field" className={field.seedType !== null ? field.seedType.name : ""}>
+        
+        {field.seedType !== null 
+            ?   <button onClick={harvest}>Recolter</button> 
+            :   <form onSubmit={plantSeeds}>
+                    <select name='seeds'>
+                        {seedTypes.map(type => <option key={type.name} value={type.name}>{type.name} : {type.buyPrice}</option>)}
```

Comment:
Configure ton éditeur pour avoir une règle verticale qui te permettra de pas dépasser 80 caractères par ligne, parce que sinon c'est un peu relou a review ;)

En l'occurence ici la ligne est trop longue du coup il y'a des chances qu'il faille indenter un peu.

https://stackoverflow.com/questions/29968499/vertical-rulers-in-visual-studio-code/52455857